File: AMS_资讯信息组件/code/server/source/com.yss.ams.sec.information.support/src/main/java/com/yss/ams/sec/information/support/modules/mp/secTransferPara/service/ISecTransferParaService.java
package com.yss.ams.sec.information.support.modules.mp.secTransferPara.service;

import java.util.List;

import com.yss.framework.api.common.co.BasePojo;
import com.yss.framework.api.mvc.biz.IServiceBus;
import com.yss.framework.api.restful.annotations.RestfulSupported;

/**
 * #42948 资讯信息管理组件化拆分
 * @author chenbo
 *2017-07-06
 */
/**
 * 证券代码转换选项服务接口
 * @author shepherd
 */
@RestfulSupported
public interface ISecTransferParaService extends IServiceBus{
	/**
	 * 修改功能选项参数
	 * @param pojoList
	 * @return
	 */
	public String updateConds(List<BasePojo> pojoList);
}

File: AMS_资讯信息组件/code/client/source/YssSecInformation.Support/Mp/SecEq/Service/ISecTransferParaService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using FAST.Common.Service.Services.Base;
using FAST.Common.Service.Pojo.Base;
using FAST.Common.Service.Attributes;


namespace YssSecInformation.Support.Mp.SecEq.Service
{
    /// <summary>
    ///  chenbo
    /// 2017-06-22
    /// #42948 资讯信息管理组件化拆分
    /// ----------------------------------
    /// 证券代码转换功能选项服务接口
    /// </summary>
    [NewServiceAttribute(ServiceType.NEW_SVC)]
    [ServiceAttribute(ServiceId = YssInformation.Support.common.ServiceIdConstant.OSGI_BASEBUSINESS, ServiceController = "com.yss.ams.sec.information.support.modules.mp.secTransferPara.controller.ISecTransferParaController", ServiceUrl = "")]
    public interface ISecTransferParaService : IServiceBus
    {
        /// <summary>
        /// 功能选项修改参数
        /// </summary>
        /// <param name="pojoList">参数列表</param>
        /// <returns>修改结果</returns>
        [MethodAttribute(Method = RequestType.POST, MethodUrl = "/updateConds")]
        string updateConds(List<BasePojo> pojoList);
    }
}
